sshu/daml_sshu.cc

```diff
@@ -0,0 +1,85 @@
+#include "eckit/config/YAMLConfiguration.h"
+#include "./eckit/filesystem/PathName.h"
+
+#include "oops/util/Logger.h"
+
+#include <torch/torch.h>
+
+// Define the Feed Forward Neural Net model.
+struct Net : torch::nn::Module {
+  Net(int inputSize, int hiddenSize, int outputSize) {
+    std::cout << inputSize << outputSize << hiddenSize << std::endl;
+    // Define the layers.
+    fc1 = register_module("fc1", torch::nn::Linear(inputSize, hiddenSize));
+    fc2 = register_module("fc2", torch::nn::Linear(hiddenSize, outputSize));
+  }
+
+  // Implement the forward pass.
```

Comment:
I thought that's been in football for over a century now?!? :-)

Reply:
Have you ever seen that error:
```
[Rank 1] Error: PE is offside.
```
It happens during deep learning when some PE's become self aware and start cheating. That's why we need to quickly switch to gpu's. I think ...

Reply:
Maybe we just need to increase the weight of the penalty